lesson21/index.js

```diff
@@ -110,5 +112,60 @@ const createTd = (usersData, keys) => {
   return fragment;
 }
 
+const createSortBtn = () => {
+  const sortBtn = createElementWithClassName("button", "sort-btn");
+  sortBtn.dataset.sortStatus = "standard";
+  sortBtn.id = "js-sort-btn";
+  return sortBtn;
+}
+
+const setClickInSortBtn = () => {
+  const standardRows = [...document.querySelector("tbody").querySelectorAll("tr")];
+  const sortBtn = document.getElementById("js-sort-btn");
+
+  sortBtn.addEventListener("click", (e) => {
+    let currentStatus = e.target.dataset.sortStatus;
+    currentStatus = switchSortStatus(currentStatus);
+    e.target.dataset.sortStatus = currentStatus;
+
+    const sortedRows = getSortedRows(currentStatus, standardRows);
+
+    const tbody = document.querySelector("tbody");
+    sortedRows.forEach((row) => {
+      tbody.appendChild(row);
+    })
+  });
+}
+
+const switchSortStatus = (status) => {
+  if (status === "standard") {
+    return "asc";
+  } else if (status === "desc") {
+    return "standard";
+  } else if (status === "asc") {
+    return "desc";
+  }
+}
+
+const getSortedRows = (status, standardRows) => {
+  if (status === "standard") {
+    return standardRows;
+  }
+  return SortById(status, standardRows);
+}
+
+const SortById = (status, standardRows) => {
```

Comment:
```suggestion
const sortById = (status, standardRows) => {
```

Reply:
Thank you.
I fixed it. -> 95aa242e64847d22c0b22ecdef07bf4c8032f176